docs/quick-start/index.md

```diff
@@ -682,19 +685,240 @@ public Server(String serverName) {
+
+Similarly to `Server`, the `Client` class of our example application wraps around
+the `io.spine.client.Client` API provided by the `spine-client` library: 
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="class Client" 
+             end="client;"?>
+```java
+public final class Client {
+
+    private final io.spine.client.Client client;
+```
+
+<p class="note">The `io.spine:spine-client` library is available
+to the example application project as a transitive dependency of
+the `io.spine:spine-server` library, which is added to the project when you do
+`spine.enableJava().server()` in your Gradle project.</p>
+
+Then, the `Client` class declares a field for keeping subscriptions to the results of a command
+execution. We'll see how this field is used in a minute. 
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="private @Nullable ImmutableSet" end="private @Nullable ImmutableSet"?>
+```java
+private @Nullable ImmutableSet<Subscription> subscriptions;
+```
+
+First of all, let's see how the `Client` instances are created.
+
+### The constructor
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="public Client(" 
+             end="    }"?>
+```java
+public Client(String serverName) {
+    this.client = inProcess(serverName)
+            .shutdownTimout(2, TimeUnit.SECONDS)
+            .build();
+}
+```
+
+Again, similarly to a `Server`, a `Client` is created using in-process connection to a named server.
+The `shutdownTimeout` parameter configures the amount of time allowed for completing current
+client-server communications before closing the client. 
+
+Now, let's review the main thing the `Client` class does, sending the `Print` command.
+
+### Sending the command
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="sendCommand() {" 
+             end="    }"?>
+```java
+public void sendCommand() {
+    String userName = System.getProperty("user.name");
+    Print commandMessage = Print.newBuilder()
+            .setUsername(userName)
+            .setText("Hello World!")
+            .vBuild();
+    this.subscriptions =
+            client.asGuest()
+                  .command(commandMessage)
+                  .observe(Printed.class, this::onPrinted)
+                  .post();
+}
+```
+
+The method does two things:
+   1. Creates a command message using the login name of the current computer user.
+   2. Posts the command, subscribing to the `Printed` event which will be generated as the result
+      of the handling the posted command.
+      
+Let's review the second thing in details.
+   
+The `client.asGuest()` call starts composing a client request. For the sake of the simplicity,
+we on do this on behalf of a user who is not logged in. A real-world application would send
```

Comment:
I also, there is "would send start sending most of the commands". Not sure which text you wanted it to be.